dotCMS/src/main/webapp/dotcms-webcomponents/p-6qksgvke.system.entry.js

```diff
@@ -0,0 +1 @@
TypeError("Generator is already executing.");while(n)try{if(r=1,i&&(o=c[0]&2?i["return"]:c[0]?i["throw"]||((o=i["return"])&&o.call(i),0):i.next)&&!(o=o.call(i,c[1])).done)return o;if(i=0,o)c=[c[0]&2,o.value];switch(c[0]){case 0:case 1:o=c;break;case 4:n.label++;return{value:c[1],done:false};case 5:n.label++;i=c[1];c=[0];continue;case 7:c=n.ops.pop();n.trys.pop();continue;default:if(!(o=n.trys,o=o.length>0&&o[o.length-1])&&(c[0]===6||c[0]===2)){n=0;continue}if(c[0]===3&&(!o||c[1]>o[0]&&c[1]<o[3])){n.label=c[1];break}if(c[0]===6&&n.label<o[1]){n.label=o[1];o=c;break}if(o&&n.label<o[2]){n.label=o[2];n.ops.push(c);break}if(o[2])n.ops.pop();n.trys.pop();continue}c=t.call(e,n)}catch(a){c=[6,a];i=0}finally{r=o=0}if(c[0]&5)throw c[1];return{value:c[0]?c[1]:void 0,done:true}}};System.register(["./p-aa15c74d.system.js"],(function(e){"use strict";var t,n,r,i,o;return{setters:[function(e){t=e.r;n=e.c;r=e.h;i=e.H;o=e.g}],execute:function(){var c=function(e){return e&&typeof e==="string"?e.split(","):[]};var a=function(e,t){if(e&&e.length&&t&&typeof t==="string"){return e.filter((function(e){var n=e.data.inode;return t.split(",").includes(n)})).map((function(e){var t=e.data;return t}))}return[]};var u=e("dot_card_view",function(){function e(e){t(this,e);this.items=[];this.selection=[];this.selected=n(this,"selected",7);this.cardClick=n(this,"cardClick",7)}e.prototype.getValue=function(){return __awaiter(this,void 0,void 0,(function(){return __generator(this,(function(e){return[2,this.selection]}))}))};e.prototype.watchItems=function(e){this.selection=a(e,this.value)};e.prototype.watchValue=function(e){this.selection=a(this.items,e)};e.prototype.render=function(){var e=this;var t=c(this.value);return r(i,null,this.items.map((function(n){return r("dot-card-contentlet",{onClick:function(t){var n=e.el.shadowRoot.querySelectorAll("dot-card-contentlet");var r=t.target;var i=false;if(t.shiftKey&&r.checked){n.forEach((function(t){if(t===r||t===e.lastChecked){i=!i}if(i){t.checked=true}}))}e.lastChecked=r},key:n.data.inode,checked:t.includes(n.data.inode),onCheckboxChange:function(t){var n=t.detail,r=n.originalTarget,i=n.data;if(r.checked){e.selection.push(i)}else{e.selection=e.selection.filter((function(e){return e.identifier!==i.identifier}))}e.value=e.selection.map((function(e){var t=e.inode;return t})).join(",");e.selected.emit(e.selection)},item:n})})))};Object.defineProperty(e.prototype,"el",{get:function(){return o(this)},enumerable:true,configurable:true});Object.defineProperty(e,"watchers",{get:function(){return{items:["watchItems"],value:["watchValue"]}},enumerable:true,configurable:true});Object.defineProperty(e,"style",{get:function(){return":host{display:grid;grid-template-columns:repeat(auto-fill,minmax(220px,1fr));grid-gap:var(--basic-padding-2)}dot-card-contentlet{height:100%}dot-card-contentlet:before{content:\"\";display:inline-block;-ms-flex:0 0 0px;flex:0 0 0;height:0;padding-bottom:calc(100%)}"},enumerable:true,configurable:true});return e}())}}}));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid assignments in operands](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5139982)